ocm_python_wrapper/cluster.py

```diff
@@ -595,7 +595,10 @@ def create_rhods_brew_config(brew_token):
                 }
             ],
         )
-        secret_data_dict = {"auths": {"brew.registry.redhat.io": {"auth": brew_token}}}
```

Comment:
`auth` key with None value is failing, therefore, the dict should be empty if `brew_token=None`.